packages/fxa-auth-server/lib/payments/paypal.ts

```diff
@@ -400,6 +403,7 @@ export class PayPalHelper {
         invoiceNumber: invoice.id,
         currencyCode: invoice.currency,
         idempotencyKey,
+        ...(ipaddress && { ipaddress }),
```

Comment:
I think just `ipaddress` here should work.

Reply:
I originally had that, but then it came through as an `undefined` which gave weird test results.